app/models/elastic_search_recommended_link.rb

```diff
@@ -23,29 +24,19 @@ def body
 
   def details
     {
-      title: @recommended_link.title,
-      link: @recommended_link.link,
-      description: @recommended_link.description,
+      title: recommended_link.title,
+      link: recommended_link.link,
```

Comment:
Good point, I've just done another rebase and extracted these all out as methods.